src/app/dataset-view/dataset.subscriptions.service.ts

```diff
@@ -28,6 +29,17 @@ export class DatasetSubscriptionsService {
         1 /*bufferSize*/,
     );
 
+    private currentTransformInputsCount$: Subject<SearchAdditionalHeaderDatasetInputs> =
+        new ReplaySubject<SearchAdditionalHeaderDatasetInputs>(1 /*bufferSize*/);
+
+    public emitCurrentTransformInputsCountChanged(count: number): void {
+        this.currentTransformInputsCount$.next({ count });
+    }
+
+    public get currentTransformInputsCountChanges(): Observable<SearchAdditionalHeaderDatasetInputs> {
+        return this.currentTransformInputsCount$.asObservable();
+    }
+
```

Comment:
This question is too low level for this service. You are putting it into a line with permissions, metadata, history, ..

Reply:
Removed